src/officehours/test_settings.py

```diff
@@ -0,0 +1,151 @@
+        self.assertIn('handlers', settings.LOGGING)
+        self.assertIn('loggers', settings.LOGGING)
+        self.assertIn('console', settings.LOGGING['handlers'])
+        self.assertIn('mail_admins', settings.LOGGING['handlers'])
+        self.assertIn('django', settings.LOGGING['loggers'])
+        self.assertIn('mozilla_django_oidc', settings.LOGGING['loggers'])
+        
+    def test_staticfiles_dirs(self):
+        self.assertIsInstance(settings.STATICFILES_DIRS, tuple)
+        
+    def test_static_files_config(self):
+        self.assertTrue(settings.STATIC_URL.startswith('/'))
+        self.assertTrue(settings.STATIC_ROOT.endswith('staticfiles'))
+
+    def test_feedback_email(self):
+        if settings.FEEDBACK_EMAIL is not None:
+            self.assertEqual(settings.FEEDBACK_EMAIL, settings.FEEDBACK_EMAIL.strip())
+            try:
+                validate_email(settings.FEEDBACK_EMAIL)
+                valid_email = True
+            except ValidationError:
+                valid_email = False
+            self.assertTrue(valid_email, "FEEDBACK_EMAIL is not a valid email")
+
+    def test_safedelete_config(self):
+        self.assertIs(settings.SAFE_DELETE_INTERPRET_UNDELETED_OBJECTS_AS_CREATED, True)
+
+    def test_drf_api_tracking_config(self):
+        self.assertIs(settings.DRF_TRACKING_ADMIN_LOG_READONLY, True)
+        self.assertIsInstance(settings.LOGGING_METHODS, list)
+        for method in settings.LOGGING_METHODS:
+            self.assertIn(method, ['POST', 'PUT', 'PATCH', 'DELETE'])
+
+    def test_email_settings(self):
+        if settings.EMAIL_HOST is not None:
+            self.assertEqual(settings.EMAIL_HOST, settings.EMAIL_HOST.strip())
+        self.assertTrue(settings.EMAIL_SUBJECT_PREFIX.startswith('['))
+        self.assertTrue(settings.EMAIL_SUBJECT_PREFIX.endswith('] '))
+
+    def test_admins_and_managers(self):
+        self.assertIsInstance(settings.ADMINS, list)
+        for admin in settings.ADMINS:
+            self.assertIsInstance(admin, tuple)
+            name, email = admin
+            self.assertTrue(name)
+            if email is not None:
+                self.assertEqual(email, email.strip())  
+            # else:  TODO: Do we need to consider when admin email is not provided?       
+            #     self.fail("Admin email cannot be None")
+
+        self.assertIsInstance(settings.MANAGERS, list)
+        self.assertTrue(set(settings.ADMINS).issubset(set(settings.MANAGERS)))
+
+    def test_google_analytics_id(self):
+        if settings.GA_TRACKING_ID is not None:
+            self.assertEqual(settings.GA_TRACKING_ID, settings.GA_TRACKING_ID.strip())
+
+    def test_channel_layers(self):
+        redis_host = settings.CHANNEL_LAYERS['default']['CONFIG']['hosts'][0][0]
+        redis_port = settings.CHANNEL_LAYERS['default']['CONFIG']['hosts'][0][1]
+        self.assertFalse('\n' in redis_host, f"The REDIS_HOST value '{redis_host}' should not contain newlines")
```

Comment:
Should we use self.assertEqual(redis_host, redis_host.strip()) instead? This is to be consistent with the other test cases.

Reply:
Changed. Thanks for noticing me to keep it consistent!